src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -71,6 +74,9 @@ static void Main(string[] args)
             TestRunSystem = ActorSystem.Create("TestRunnerLogging");
             SinkCoordinator = TestRunSystem.ActorOf(Props.Create<SinkCoordinator>(), "sinkCoordinator");
 
+            var tcpLogger = TestRunSystem.ActorOf<TcpLoggingServer>();
+            TestRunSystem.Tcp().Tell(new Tcp.Bind(tcpLogger, new IPEndPoint(IPAddress.Any, 5678)));
```

Comment:
Does it need to be a hardcoded port? Maybe a config value or program parameter would be a better idea?